src/SprykerEco/Yves/Heidelpay/Form/CreditCardSecureSubForm.php

```diff
@@ -8,26 +8,54 @@
 namespace SprykerEco\Yves\Heidelpay\Form;
 
 use Generated\Shared\Transfer\HeidelpayCreditCardPaymentTransfer;
+use Spryker\Yves\StepEngine\Dependency\Form\AbstractSubFormType;
 use Spryker\Yves\StepEngine\Dependency\Form\SubFormInterface;
+use Spryker\Yves\StepEngine\Dependency\Form\SubFormProviderNameInterface;
 use SprykerEco\Shared\Heidelpay\HeidelpayConfig;
 use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\Validator\Constraint;
 use Symfony\Component\Validator\Constraints\NotBlank;
 
-class CreditCardSecureSubForm extends AbstractHeidelpaySubForm
+class CreditCardSecureSubForm extends AbstractSubFormType implements SubFormInterface, SubFormProviderNameInterface
```

Comment:
Again I don't get the idea with getting rid of AbstractHeidelpaySubForm in a way like this

Reply:
Answered above.